Data/Primitive/ByteArray.hs

```diff
@@ -118,7 +118,7 @@ instance TH.Lift ByteArray where
 #if __GLASGOW_HASKELL__ >= 802
       pinned = isByteArrayPinned ba
 #else
-      pinned = False
+      pinned = True
```

Comment:
Uh.

> This function is only available when compiling with GHC 8.2 or newer.

on `isByteArrayPinned` is almost impossible to spot.

`isByteArrayPinned#` in `base` doesn't have `@since` annotations either.